Add link to new workload set documentation (#9483)
dotnet/core: release-notes/9.0/preview/preview7/sdk.md

```terminal
> dotnet workload --info
 Workload version: 9.0.100-preview.7.24414.1
Configured to use loose manifests when installing new manifests.
 [aspire]

>    Installation Source: SDK 8.0.400

Reviewer: is this a valid result for .NET 9 preview 7?

Reviewer 2: If the aspire workload was last installed when using 8.0.400-band SDKs, then yes - the next time a workload update happened (via `dotnet workload install` or `dotnet workload update`) I'd expect this to change to align with the in-use SDK's feature band.

   Manifest Version:    8.1.0/8.0.100
   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.aspire\8.1.0\WorkloadManifest.json
   Install Type:        FileBased
```

> In this example, I am in 'manifests' mode, which is what we call the current mode of managing workloads.

Reviewer: not your text but maybe we should change the text here to not use I?

```suggestion
This example uses the 'manifests' mode, which is what we call the current mode of managing workloads.
```
